.circleci/config.yml

```diff
@@ -77,10 +77,10 @@ jobs:
       - run:
           name: 🏗️ build dash
           command: |
-            . venv/bin/activate && pip install --upgrade -e . --quiet && mkdir packages
+            . venv/bin/activate && mkdir packages
             # build main dash
             git clone --depth 1 https://github.com/plotly/dash.git dash-main
-            cd dash-main && pip install -e .[dev,testing] && python setup.py sdist && mv dist/* ../packages/
+            cd dash-main && pip install -e .[dev,testing] --quiet && python setup.py sdist && mv dist/* ../packages/
```

Comment:
I think our intention is to mute the progress bar,  ` --progress-bar off`
at least for now, we need to check the pip install logs from time to time

Reply:
https://github.com/plotly/dash-core-components/pull/659/commits/56093b14ebd5ca7170da85b61fa3cf781e686c8c